frontend/src/main/web/src/app/pages/approver/dashboard/shared/pipes/trip-filter.pipe.ts

```diff
@@ -0,0 +1,28 @@
+import {Pipe, PipeTransform} from '@angular/core';
+import {PendingTrip} from "../model/pending-trip.model";
+
+@Pipe({
+  name: 'itemsFilter'
+})
+export class ItemsFilter implements PipeTransform {
+
+  transform(trip: PendingTrip[], currentFilterCriteria: string, currentFilterContent: string): PendingTrip[] {
+
+    if (currentFilterContent === 'In draft'.toLowerCase()) {
+      currentFilterContent = '1';
```

Comment:
Right now service and trip endpoints give string in a different formant - "DRAFT" instead of "1". Please check if your code works. Unless you are using your own endpoints, it shouldn't. If you are using your own endpoints, well good job on you.

Reply:
When I was implementing BE part, I use my own SQL. I decided not to use other code because it doesn't fit to my task.